docs-examples/example-groovy/build.gradle

```diff
@@ -2,8 +2,13 @@ plugins {
     id "groovy"
 }
 
+repositories {
+    mavenCentral()
+}
+
 dependencies {
     testImplementation project(":openapi")
+    testImplementation platform("io.micronaut:micronaut-bom:$micronautVersion")
```

Comment:
I needed to add the repositories and the `platform` dependency (and the same in the java and kotlin projects) to be able to resolve the dependencies. Before the changes this wasn't necessary so I'm not sure if I'm missing something.

Reply:
This is probably because before configuration was applied unconditionally on all subprojects, which isn't the case anymore.